src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,6 +1,25 @@
 import React from 'react';
+
 import './MoviesList.scss';
 
-export const MoviesList: React.FC = () => (
-  <>Put the list here</>
+import { Movie } from '../../types/Movie';
+import { MovieCard } from '../MovieCard';
+
+type Props = {
+  movies: Movie[];
+};
+
+export const MoviesList: React.FC<Props> = ({ movies }) => (
+  <ul className="movies">
+    {movies.map(movie => (
+      <li className="card" data-cy="Movie" key={movie.imdbId}>
+        <MovieCard
+          title={movie.title}
+          description={movie.description}
+          imgUrl={movie.imgUrl}
+          imdbUrl={movie.imdbUrl}
+        />
```

Comment:
It's better to pass the movie to MovieCard and do destructure in the MovieCard. As in the checklist 'The MoviesList should pass a movie to the MovieCard.'

Reply:
Yep, thats a bad practice not to read the task carefully)
Thanks!